test/aliases.js

```diff
@@ -0,0 +1,68 @@
+import { versions, cwd as getCwd, chdir } from 'process'
+
+import test from 'ava'
+
+import { resolveVersionRangeAlias } from '../src/aliases.js'
+import normalizeNodeVersion from '../src/main.js'
+
+const resolveInFolder = (versionRange, folder) =>
+  normalizeNodeVersion(versionRange, {
+    cwd: `${__dirname}/fixtures/${folder}`,
+  })
+
+test.serial('Option cwd defaults to the current directory', async (t) => {
+  const currentCwd = getCwd()
+  chdir(`${__dirname}/fixtures/nvmrc-project`)
+
+  try {
+    const versionRange = await normalizeNodeVersion('.')
+    t.is(versionRange, '12.14.1')
+  } finally {
+    chdir(currentCwd)
+  }
+})
+
+test('Resolve . with node-version pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'node-version-project')
+  t.is(versionRange, '12.12.0', 'nvmrc version not resolved')
+})
+
+test('Resolve . with nvmrc pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'nvmrc-project')
+  t.is(versionRange, '12.14.1', 'nvmrc version not resolved')
+})
+
+test('Resolve . with nave pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'nave-project')
+  t.is(versionRange, '12.16.1', '.nave version not resolved')
+})
+
+test('Resolve . in mixed project pseudo version, .node-version having precedence over .nvmrc', async (t) => {
+  const versionRange = await resolveInFolder('.', 'mixed-project')
+  t.is(versionRange, '12.12.0', 'not resolved to .node-version')
+})
+
+test('Resolve . default to process.version if no version file found', async (t) => {
+  const versionRange = await normalizeNodeVersion('.', {
+    cwd: `${__dirname}/../src`,
+  })
+  t.is(versionRange, versions.node, 'not resolved to .node-version')
+})
+
+test('Resolve - current node pseudo version', async (t) => {
+  const versionRange = await normalizeNodeVersion('_')
+
+  t.is(versionRange, versions.node, 'current version not resolved correctly')
+})
+
+test('Transfer node version that are not aliased [fully resolved]', async (t) => {
+  const versionRange = await resolveVersionRangeAlias('v12.12.0')
+
+  t.is(versionRange, 'v12.12.0', 'current version not resolved correctly')
+})
+
+test('Transfer node version that are not aliased [major version]', async (t) => {
+  const versionRange = await resolveVersionRangeAlias('12')
```

Comment:
```suggestion
  const versionRange = await normalizeNodeVersion('12')
```

Reply:
This one was really left on purpose as I mentioned [there](https://github.com/ehmicky/normalize-node-version/pull/1#issuecomment-609070093).

And ensure no alias resolving took place (This would more make sense, when handling the nvm alias from https://github.com/AdrieanKhisbe/normalize-node-version/tree/support-nvm-aliases)

Reply:
Which feature we are testing here from an end-user perspective?

If I understand it correctly, we are testing the case where no alias was used. In which case, the original version should be passed as is. Since the user does not see that intermediate state and only sees the finally resolved version, it seems to me those two tests are already covered by the tests in `test/main.js`, i.e. could be removed?

Reply:
Hello @ehmicky :), Sorry I didn't see that message. :/

To me these tests are not testing from a specific end-user perspective, indeed this is already covered by main.

It's a test to enforce on `resolveVersionRangeAlias` assumed behavior if you can resolve alias, just pass along version.

As it's more related to second phase of nvm alias support I'll remove them from PR, this so we can move forward.
 
-----
Ps: here is the relevant line I have in mind that make me think we might need test if we extend `resolveVersionRangeAlias` with more features:
- https://github.com/AdrieanKhisbe/normalize-node-version/blob/support-nvm-aliases/src/aliases.js#L66-L67
- https://github.com/AdrieanKhisbe/normalize-node-version/blob/support-nvm-aliases/test/aliases.js#L58-L64